src/com/mambu/apisdk/services/SavingsService.java

```diff
@@ -248,6 +248,10 @@ public SavingsAccount undoApproveSavingsAccount(String accountId, String notes)
 	/**
 	 * Get savings transactions by specifying filter constraints
 	 * 
+	 * Note: This method is deprecated, you may use the getSavingsTransactionsWithFullDetails in order to obtain savings
+	 * transaction with full (custom field included) details or getSavingsAccountsWithBasicDetails to obtain the savings

```

Comment:
transaction**s**

Reply:
also, in order to be consistent with the comments from LoansService can have 

> full (custom field included) details

 =>  
full details (custom field**s** included)
> 

Please note also the "s" after "field"

Reply:
fixed, thanks!